Source/WebCore/Modules/webxr/WebXROpaqueFramebuffer.cpp

```diff
@@ -77,6 +77,16 @@ WebXROpaqueFramebuffer::~WebXROpaqueFramebuffer()
         m_multisampleColorBuffer.release(*gl);
         m_resolvedFBO.release(*gl);
         m_context.deleteFramebuffer(m_framebuffer.ptr());
+    } else {
+        // The GraphicsContextGL is gone, so disarm the GCGLOwned objects so
+        // their destructors don't assert.
+#if USE(IOSURFACE_FOR_XR_LAYER_DATA)
+        m_opaqueTexture.leakObject();
```

Comment:
Are these actually memory leaks? If so, do we need to be concerned?

Reply:
I used `leakObject` in keeping with `Ref` which has `leakRef` that returns the internal raw pointer and resets. Given that we no longer have the `GraphicsContextGL` to free the allocation, there's not too much we can do. As I describe in the commit message, I want to keep the ASSERT behavior in the destructor to catch when people forget to release these objects.